src/client/javascript/helpers/default-application.js

```diff
@@ -198,8 +203,9 @@
    * @param   {OSjs.VFS.File}       file        File
    * @param   {OSjs.Core.Window}    win         Window reference
    * @param   {Boolean}             saveAs      SaveAs ?
+   * @param   {CallbackDialog}      cb          Called after the user closed the dialog
    */
-  DefaultApplication.prototype.saveDialog = function(file, win, saveAs) {
+  DefaultApplication.prototype.saveDialog = function(file, win, saveAs, cb) {
```

Comment:
You are right, I usually write ES6 with default values for arguments, I removed the default arg but forgot to handle it. Thank you for the heads up.